go.mod

```diff
@@ -69,6 +69,7 @@ require (
 	github.com/multiformats/go-multistream v0.0.1
 	github.com/opencontainers/go-digest v1.0.0-rc1 // indirect
 	github.com/opencontainers/image-spec v1.0.1 // indirect
+	github.com/otiai10/copy v1.0.1
```

Comment:
I looked over this package and it seems legit, not much to it and does the right thing simply and seems to be properly done.